src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -520,6 +522,51 @@ public String handle(
 			return gson.toJson(result);
 		}
 	}
+	
+	@Path("/api/v1/setSystem")
+	public class SetSystemEndpoint implements Route {
+		@POST
+		@Produces({ MediaType.APPLICATION_JSON })
+		@Operation(
+			summary = "System commands details", 
+			description = "Perform some system wide commands.", 
+			operationId = "systemCommand", 
+			tags = {"SDK" }, 
+			requestBody = @RequestBody(
+				description = "Command details", 
+				content = {
+					@Content(
+						mediaType = "application/json", 
+						schema = @Schema(implementation = JSONObject.class))
+				}, 
+				required = true
+			), 
+			responses = {
+				@ApiResponse(
+					responseCode = "200", 
+					description = "Successful command execution", 
+					content = @Content(
+						// TODO: Provide a specific class as value of Schema.implementation
+						schema = @Schema(implementation = Object.class)
+					)
+				),
+				@ApiResponse(responseCode = "400", description = "Bad Request")
+			}
+		)
+		@Override
+		public String handle(
+				@Parameter(hidden = true) Request request,
+				@Parameter(hidden = true) Response response
+		) {
+			try {
+				JSONObject obj = gson.fromJson(request.body(), JSONObject.class);
```

Comment:
Don't mix `JSONObject` and `Gson` - they are two different libraries (the gson library's version is called `JsonObject`, which may have been confusing). You can choose to use either library here.

```java
// gson
JsonObject obj = gson.fromJson(request.body(), JsonObject.class);

// org.json
JSONObject obj = new JSONObject(request.body());
```